dali/operators/reader/loader/webdataset_loader.h

```diff
@@ -32,8 +31,35 @@ namespace dali {
 namespace detail {
 namespace wds {
 
-const std::string kCurrentIndexVersion = "v1.2";  // NOLINT
-const std::unordered_set<std::string> kSupportedIndexVersions = {"v1.1", kCurrentIndexVersion};
+constexpr int ParseIndexVersion(const char* index_version) {
+  const char* it = nullptr;
```

Comment:
What if the data is invalid?

Reply:
Yes, I was talking about the 2) case as this parses user-provided input. Malformed or even malicious input should not crash the application but throw an appropriate error.

> I'll add the function that roughly checks the syntax of the index

Maybe a regexp check?
And I would add tests for that as well.

Reply:
Done